.nix/hs-notmuch.nix

```diff
@@ -1,10 +1,15 @@
 { mkDerivation, base, bytestring, c2hs, deepseq, mtl, notmuch
-, profunctors, stdenv, tagged, talloc, text, time
+  , profunctors, stdenv, tagged, talloc, text, time, fetchFromGitHub
 }:
 mkDerivation {
   pname = "notmuch";
   version = "0.2.0.0";
-  sha256 = "feeafa5bdc00c1036755c92d72e7b69cb388ae9bab5064410655007cc8216ff0";
```

Comment:
This obviously needs adjusting once you merged the changes, including the setting the upper version bound in the cabal file.